browse/static/js/coreRecommender.js

```diff
@@ -0,0 +1,62 @@
+function getCORERecCookie() {
+    var coreCookie = "arxiv_core";
```

Comment:
I recommend to use a global variable for this string to be able to change it in the single point. Also, I would explicitly call it `arxiv_core_recommender` because at some point we may integrate another service.